repobee_sanitizer/_sanitize_repo.py

```diff
@@ -136,6 +142,14 @@ def _sanitize_to_target_branch(
         )
 
 
+def _check_repo_state(repo_root) -> str:
+    repo = git.Repo.init(repo_root)
+    if repo.head.commit.diff():
+        return "There are uncommited staged files in the repo"
+    else:
+        return None
```

Comment:
Very tiny comment, but in general this looks like so in RepoBee:
```suggestion
    repo = git.Repo.init(repo_root)
    if repo.head.commit.diff():
        return "There are uncommited staged files in the repo"
    return None
```
I think this makes it more clear that `return None` is the "default" case. This is my personal opinion and I'm not saying it's better, but it's the style that is used throughout RepoBee, and so I think we should keep to it.